bin/console

```diff
@@ -5,7 +5,7 @@ use Symfony\Component\Console\Application;
 use MadeByDenis\Commands\CleanCodeCoverage;
 
 if (!in_array(PHP_SAPI, ['cli', 'phpdbg', 'embed'], true)) {
-    echo 'Warning: The console should be invoked via the CLI version of PHP, not the '.PHP_SAPI.' SAPI'.PHP_EOL;
+    error_log(sprintf('Warning: The console should be invoked via the CLI version of PHP, not the %s SAPI', PHP_SAPI));
```

Comment:
If I'm not mistaken, this won't actually show the message to the user if (for whatever reason) the console command is tried to be used in the app directly instead of in the terminal? The intent is to notify the person not to do that 😄 

Maybe a return or error thrown should be even better in this case 🤔

Reply:
I see.
Making software for non-experts is still highly abnormal for me.

Reply:
> Making software for non-experts is still highly abnormal for me.

I see you haven't worked in an agency 😂